p521/src/arithmetic/util.rs

```diff
@@ -30,3 +32,47 @@ pub(crate) const fn u64x9_to_u32x18(w: &[u64; 9]) -> [u32; 18] {
 
     ret
 }
+
+/// Converts the saturated representation [`U576`] into a 528bit array. Each
+/// word is copied in little-endian.
+pub const fn uint_to_le_bytes_unchecked(w: U576) -> [u8; 66] {
```

Comment:
We should probably add some `const fn` serializers upstream in `crypto-bigint` that could handle this sort of thing, but this is fine for now.